waku/waku_archive/driver/sqlite_driver/queries.nim

```diff
@@ -117,39 +105,15 @@ proc createOldestMessageTimestampIndex*(db: SqliteDatabase): DatabaseResult[void
     )
   return ok()
 
-proc createHistoryQueryIndexQuery(table: string): SqlQueryStr =
-  "CREATE INDEX IF NOT EXISTS i_query ON " & table &
-    " (contentTopic, pubsubTopic, storedAt, id);"
-
-proc createHistoryQueryIndex*(db: SqliteDatabase): DatabaseResult[void] =
-  let query = createHistoryQueryIndexQuery(DbTable)
-  discard
-    ?db.query(
-      query,
-      proc(s: ptr sqlite3_stmt) =
-        discard
-      ,
-    )
-  return ok()
```

Comment:
Why this can be removed?

Reply:
A performance review of Store v3 is planned and will guide the creation of new indexes.